beacon-chain/core/validators/validator_test.go

```diff
@@ -132,7 +132,7 @@ func TestSlashValidator_OK(t *testing.T) {
 	state, err := v1.InitializeFromProto(base)
 	require.NoError(t, err)
 
-	slashedIdx := types.ValidatorIndex(2)
+	slashedIdx := types.ValidatorIndex(3)
```

Comment:
This seems like a strange change. Could you explain?

Reply:
In the test we calculate the proposer index using `proposer, err := helpers.BeaconProposerIndex(context.Background(), state)`. Internally it uses the state for randomness calculations. After changing the configuration to minimal, the calculated proposer changed to `2`, which clashed with `slashedIdx`.